### PR TITLE
[feature][website] release notes for pulsar manager 0.3.0

### PR DESCRIPTION
**Pull request by urfreespace**
### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [x] `no-need-doc` 
(Please explain why)
  
- [ ] `doc` 
(Your PR contains doc changes)

- [ ] `doc-added`
(Docs have been already added)

### CODE REVIEW

File: site2/website/pulsar-manager-release-notes.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 ## Apache Pulsar Manager
 
+### 0.3.0 &mdash; 2022-05-25 <a id="0.3.0"></a>
+* Fixed mysql schema field token error [PR-339](https://github.com/apache/pulsar-manager/pull/339)
```
Comment (Anonymitaet) [inline site2/website/pulsar-manager-release-notes.md:4]:
```suggestion
* Fixed MySQL schema field token error [PR-339](https://github.com/apache/pulsar-manager/pull/339)
```

1. `Fix` of `Fixed`? Keep tense consistent overall. For example, use past tense or simple tense for all verbs. Please update all occurrences. 

3. Capitalize proper nouns correctly.  https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.je5thtkj42cb

File: site2/website/pulsar-manager-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+* Fixed topics List page, search Topics bug [PR-342](https://github.com/apache/pulsar-manager/pull/342)
```
Comment (Anonymitaet) [inline site2/website/pulsar-manager-release-notes.md:5]:
```suggestion
* Fix the bugs in topics list page and search topics [PR-342](https://github.com/apache/pulsar-manager/pull/342)
```
Do you mean this？

File: site2/website/pulsar-manager-release-notes.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+* Fixed pulsar manager's swagger dose not work. [PR-369](https://github.com/apache/pulsar-manager/pull/369)
```
Comment (Anonymitaet) [inline site2/website/pulsar-manager-release-notes.md:6]:
```suggestion
* Fix the issue of Swagger does not work. [PR-369](https://github.com/apache/pulsar-manager/pull/369)
```

File: site2/website/pulsar-manager-release-notes.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+* Add offload threshold input [PR-375](https://github.com/apache/pulsar-manager/pull/375)
+* Add dashboard service for aggregated information collection [PR-326](https://github.com/apache/pulsar-manager/pull/326)
+* Multi bookie cluster Support [PR-388](https://github.com/apache/pulsar-manager/pull/388)
```
Comment (Anonymitaet) [inline site2/website/pulsar-manager-release-notes.md:9]:
```suggestion
* Add multi bookie cluster support [PR-388](https://github.com/apache/pulsar-manager/pull/388)
```
do you mean this?

File: site2/website/pulsar-manager-release-notes.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
...
+* Fixed the bug that admin roles cannot be deleted. [PR-397](https://github.com/apache/pulsar-manager/pull/397)
```
Comment (Anonymitaet) [inline site2/website/pulsar-manager-release-notes.md:10]: Use a period or not? please keep consistent

File: site2/website/pulsar-manager-release-notes.md [hunk 6]
```diff
@@ -1,5 +1,19 @@
...
+* Add filterable to el-select enables filtering for tenant/namespace/topic [PR-440](https://github.com/apache/pulsar-manager/pull/440)
```
Comment (Anonymitaet) [inline site2/website/pulsar-manager-release-notes.md:11]: filterable is an adjective. filterable that? filterable condition?

File: site2/website/pulsar-manager-release-notes.md [hunk 7]
```diff
@@ -1,5 +1,19 @@
...
+* Updated to fix Log4J security vulnerabilities [PR-438](https://github.com/apache/pulsar-manager/pull/438)
+* Allow user to assign tenant as resource to role [PR-436](https://github.com/apache/pulsar-manager/pull/436)
+* Add support for casdoor [PR-446](https://github.com/apache/pulsar-manager/pull/446)
```
Comment (Anonymitaet) [inline site2/website/pulsar-manager-release-notes.md:14]:
```suggestion
* Add support for Casdoor [PR-446](https://github.com/apache/pulsar-manager/pull/446)
```

File: site2/website/pulsar-manager-release-notes.md (hunk 1 above)
Comment (urfreespace) [reply site2/website/pulsar-manager-release-notes.md]: fixed, thanks.

File: site2/website/pulsar-manager-release-notes.md (hunk 2 above)
Comment (urfreespace) [reply site2/website/pulsar-manager-release-notes.md]: fixed, thanks.

File: site2/website/pulsar-manager-release-notes.md (hunk 3 above)
Comment (urfreespace) [reply site2/website/pulsar-manager-release-notes.md]: fixed, thanks.

File: site2/website/pulsar-manager-release-notes.md (hunk 4 above)
Comment (urfreespace) [reply site2/website/pulsar-manager-release-notes.md]: fixed, thanks.

File: site2/website/pulsar-manager-release-notes.md (hunk 5 above)
Comment (urfreespace) [reply site2/website/pulsar-manager-release-notes.md]: fixed, thanks.

File: site2/website/pulsar-manager-release-notes.md (hunk 7 above)
Comment (urfreespace) [reply site2/website/pulsar-manager-release-notes.md]: fixed, thanks.

File: site2/website/pulsar-manager-release-notes.md (hunk 6 above)
Comment (urfreespace) [reply site2/website/pulsar-manager-release-notes.md]: fixed, thanks.

**Review by tuteng — APPROVED** [review]

**Pull request merged**